mapping/mapping.go

```diff
@@ -85,6 +85,12 @@ func (h helperImpl) createConsumer(entry config.RabbitEntry) consumer.Client {
 		"consumerName": entry.Name}).Info("Creating consumer")
 	switch entry.Type {
 	case rabbitmq.Type:
+		if entry.ConnectionURLEnvKey != "" {
```

Comment:
is this needed? couldnt we just read the env var and write it to the config file before starting?